CMakeLists.txt

```diff
@@ -308,21 +307,8 @@ elseif (GUILE22_FOUND) # found guile-2.2
   message(STATUS "Using guile-2.2.x")
   find_program (GUILE_EXECUTABLE NAMES guile2.2 guile)
 
-elseif (GUILE2_FOUND) # found guile-2.0
-  add_definitions (-DHAVE_GUILE20)
-  set(HAVE_GUILE2 TRUE)
-  set(GUILE_EFFECTIVE_VERSION 2.0)
-  set(GUILE_INCLUDE_DIRS ${GUILE2_INCLUDE_DIRS})
-  set(GUILE_LDFLAGS ${GUILE2_LDFLAGS})
-  find_program (GUILD_EXECUTABLE NAMES guild2.0 guild)
-  if (NOT GUILD_EXECUTABLE)
-    message (SEND_ERROR "The guild executable was not found, but is required. Please set GUILD_EXECUTABLE.")
-  endif()
-  message(STATUS "Using guile-2.0.x")
-  find_program (GUILE_EXECUTABLE NAMES guile2.0 guile)
-
 else()
-  message (FATAL_ERROR "Neither guile 3.0, guile 2.2, nor guile 2.0 were found GnuCash can't run without one of them. Ensure that one is installed and can be found with pkg-config.")
+  message (FATAL_ERROR "Neither guile 3.0, nor guile 2.2.1 were found GnuCash can't run without one of them. Ensure that one is installed and can be found with pkg-config.")
```

Comment:
… were found. GnuCash can't …

Reply:
And `Neither guile 3.0 nor guile 2.2.1+ were found.` A comma is used only when there are more than two elements in the list.